<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUserListDialog</class>
 <widget class="QWidget" name="QUserListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>30</number>
   </property>
   <item>
    <widget class="QTableWidget" name="userTableWidget">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Last Login</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetPasswordPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
